notebook2/notebook/notebook.py

```diff
@@ -315,6 +347,51 @@ def wait_websocket(ws):
     log.info(f'notification sent to user for {svc_name} {pod_name}')
 
 
+@app.route('/auth0-callback')
+def auth0_callback():
+    # https://github.com/auth0-samples/auth0-python-web-app/commit/d048d6497caa714c52e8411a5f37500787e37305
+    auth0.authorize_access_token()
```

Comment:
It seems like the appropriate link would be to the auth0 docs: https://auth0.com/docs/quickstart/webapp/python, but, IMO, we can expect readers of this code to familiarize themselves with the basic docs of the libraries we use.

Reply:
This was mostly a comment to show users how to extract the access token, which is effectively undocumented.I found it not at all clear, from either auth lib or auth0. In general, I’ve found the reliance on a 3rd party package (relative to auth0) unpleasant (for instance, try changing the request_type to something other than ‘code’).

Reply:
Maybe we're talking about different things? The auth0 quickstart clearly documents this pattern. Which method has a `request_type` parameter? I don't see that in the authlib docs.

Reply:
I had wanted to document how to extract out the access token from the response, which I don't think is documented. For the grant that we're using it isn't really necessary, so I'll drop the comment.